src/NzbDrone.Core/Localization/LocalizationService.cs

```diff
@@ -133,13 +133,15 @@ private string GetLanguageFileName()
 
             await CopyInto(dictionary, baseFilenamePath).ConfigureAwait(false);
 
-            if (culture.Contains('_'))
+            if (culture.Contains('_') && !File.Exists(alternativeFilenamePath))
             {
                 var languageBaseFilenamePath = Path.Combine(prefix, GetResourceFilename(culture.Split('_')[0]));
                 await CopyInto(dictionary, languageBaseFilenamePath).ConfigureAwait(false);
             }
-
-            await CopyInto(dictionary, alternativeFilenamePath).ConfigureAwait(false);
+            else
```

Comment:
This breaks full fallback logic.

Ex. Currently it fills in any translations from the base language and then overwrites with any translations from the dialect. So for cases like pt and pt_BR. if pt has more strings, they get used as well for a fuller experience.

The easiest thing to do here is change the error to a log or something as Bogdan eluded to in the issue.

Reply:
try/catch and a log warning it is then.... or should we make it debug?

Reply:
Debug or trace is probably fine.